testing/jormungandr-testing-utils/src/wallet/committee.rs

```diff
@@ -208,4 +216,46 @@ impl PrivateVoteCommitteeDataManager {
     pub fn member_public_keys(&self) -> Vec<MemberPublicKey> {
         self.data.values().map(|x| x.member_public_key()).collect()
     }
+
+    pub fn decrypt_tally(&self, vote_plan_status: &VotePlanStatus) -> DecryptedPrivateTally {
+        let encrypted_tally = vote_plan_status
+            .proposals
+            .iter()
+            .map(|proposal| {
+                let tally_state = proposal.tally.as_ref().unwrap();
+                let encrypted_tally = tally_state.private_encrypted().unwrap().0.clone();
+                let max_votes = tally_state.private_total_power().unwrap();
+                (encrypted_tally, max_votes)
+            })
+            .collect::<Vec<_>>();
+
+        let absolute_max_votes = encrypted_tally
+            .iter()
+            .map(|(_encrypted_tally, max_votes)| *max_votes)
+            .max()
+            .unwrap();
+        let table =
+            chain_vote::TallyOptimizationTable::generate_with_balance(absolute_max_votes, 1);
```

Comment:
It's better if you leave the default balance here if you are not sure there will always be only one proposal.

Reply:
Very good catch! This is a copy-paste issue. so basically instead of 1 there should be number of proposals ?

Reply:
the default is 2, but I suggest to use `generate(...)` that take care of this if we ever want to change that default. `generate_with_balance` should be used only if you have to decrypt either only 1 proposal or a lot of them.